src/vpn-api/utils/service.ts

```diff
@@ -70,9 +79,7 @@ class ServiceInstance {
 			})
 			.return(true)
 			.catch(err => {
-				captureException(err, 'Failed to send a heartbeat to the API', {
```

Comment:
You've removed the message info from all the captures, I think it'd make sense to keep them so we get more info about where an error is occurring rather than just the stack trace (which depending on the error might be completely useless due to happening in an async callback)

Reply:
Added fingerprints as discussed.